docs/content/en/docs/clustermgmt/cluster-backup-restore/backup-cluster.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Backup cluster"
+linkTitle: "Backup cluster"
+weight: 20
+aliases:
+    /docs/tasks/cluster/cluster-backup-restore/backup-cluster/
+description: >
+  How to backup your EKS Anywhere cluster
+---
+
+We strongly advise performing regular cluster backups of all the EKS Anywhere clusters under your management. This ensures that you always have an up-to-date cluster state available for restoration in case the cluster experiences issues or becomes unrecoverable. This document outlines the steps for creating the two essential types of backups required for the [EKS Anywhere cluster restore process]({{< relref "./restore-cluster" >}}).
+
+## Etcd backup
+
+For optimal cluster maintenance, it is crucial to perform regular etcd backups on all your EKS Anywhere management and workload clusters. In the event of a cluster upgrade failure, make sure to **always** execute the etcd backup beforehand. To create an etcd backup for your cluster, follow the guidelines provided in the [External etcd backup and restore]({{< relref "../etcd-backup-restore/etcdbackup" >}}) section.
+
+
+## Cluster API backup
+
+Since cluster failures primarily occur following unsuccessful cluster upgrades, EKS Anywhere takes the proactive step of automatically creating backups for the Cluster API objects that capture the states of both the management cluster and its workload clusters. These backups are stored within the management cluster folder, where the upgrade command is initiated from the Admin machine, and are generated before each upgrade process.
```

Comment:
Just want to confirm, does the workload cluster upgrade also take a backup of all other clusters including management?
Also, I think adding the folder layout here might be beneficial. Something like this:
```
my-mgmt-cluster/
├── cluster-state-backup-2023-10-11T02_55_56        <------ Folder with a backup of CAPI objects
├── my-mgmt-cluster-eks-a-cluster.kubeconfig
├── my-mgmt-cluster-eks-a-cluster.yaml
└── generated
```